time stamp
float64 roll
float64 pitch 
float64 head
float64 var_roll
float64 var_pitch 
float64 var_head